### DISCUSSION TITLE
Tag 3.13.2 is missing several archs

### DISCUSSION THREAD

**Discussion by WyriHaximus** [docker-library/rabbitmq#700] [Q&A] [answered]
Compared to tag [`3.13.1`](https://hub.docker.com/layers/library/rabbitmq/3.13.1/images/sha256-b9b66a6e49e5d7ddc20ab130d1f5b3c902f166270e43bd0d85fe5a99ba8e3c96?context=explore), tag [`3.13.2`](https://hub.docker.com/layers/library/rabbitmq/3.13.2/images/sha256-83166033394f6b70fa58520dc9d63d22d393c96ce879b3a886b774d0055aaa3a?context=explore) is missing the `arm/v7` and `arm64` archs.

**Comment by lukebakken** [comment]
This happens on every release, it seems.

https://github.com/search?q=repo%3Adocker-library%2Frabbitmq+missing&type=issues

Please be patient. There may be an actual issue, I'm not sure, but 3.13.2 was released pretty recently.

@yosifkit @tianon - anything different this time?

**Reply by WyriHaximus** [reply, marked as answer]
@lukebakken I'm aware it can take up to a day, wasn't aware it could take up to 4 days. They came through 40 minutes ago. ([3.13.2-management](https://hub.docker.com/layers/library/rabbitmq/3.13.2-management/images/sha256-11a89d854912f22a317b51c6b987684fa0bbedb1f9e3edf2dbcd48bff0c4ebd9?context=explore) seems to still miss them.) If in RabbitMQ's case, it can take up to 5 days extra I'll keep that in mind 👍 .

**Reply by tianon** [reply]
I think the only thing unusual in this instance was that it was merged pretty close to https://github.com/docker-library/official-images/pull/16704 (and merging popular base images like Debian and Ubuntu usually clogs the queues for longer than usual, unfortunately).

**Reply by yosifkit** [reply]
This time it was delayed by a bug we run into with buildkit and we weren't able to run the fix until after the weekend.

https://doi-janky.infosiftr.net/job/wip/job/new/job/build-arm64v8/20351/console:

```console
#19 [stage-3 2/9] COPY --from=openssl-builder /opt/openssl /opt/openssl
#19 CACHED

#20 [stage-3 8/9] COPY docker-entrypoint.sh /usr/local/bin/
#20 CACHED

#21 docker-image://docker.io/docker/scout-sbom-indexer:1-doi@sha256:630fb661ff8194ea0728758b003c9d8fd22908f0fa0952cef2196c826e8c4913
#21 resolve docker.io/docker/scout-sbom-indexer:1-doi@sha256:630fb661ff8194ea0728758b003c9d8fd22908f0fa0952cef2196c826e8c4913 0.0s done
#21 DONE 0.0s

#22 [linux/arm64] generating sbom using docker.io/docker/scout-sbom-indexer:1-doi@sha256:630fb661ff8194ea0728758b003c9d8fd22908f0fa0952cef2196c826e8c4913
#22 CACHED

#23 exporting to oci image format
#23 exporting layers done
#23 exporting manifest sha256:89c5501c6585789984fa164ef9f3ec44c7925d0568cdc22121970d70df2d9356 done
#23 exporting config sha256:269d27f4521a93f11d4b177bcd2cd441cb2e0368f4fb921f4ea24e3c59dbff5b done
#23 ERROR: content digest sha256:d5a2ad729c09fbfdb259ae764f92188efc67a615ffde9bb34a46298d1edf3cd6: not found
------
 > exporting to oci image format:
------
ERROR: failed to solve: content digest sha256:d5a2ad729c09fbfdb259ae764f92188efc67a615ffde9bb34a46298d1edf3cd6: not found
```

**Reply by WyriHaximus** [reply]
Thanks for the updates, and thank you for all your work 👍
